state/atoms/measureBundle.ts

```diff
@@ -3,6 +3,7 @@ import { atom } from 'recoil';
 interface measureBundleStateType {
   name: string;
   content: fhir4.Bundle | null;
+  valueSetsMap: { [key: string]: string } | null;
```

Comment:
A benefit that this also gives us is that we don't have to explicity compute the valueset map in the `extractMeasureBundle` function, it will just auto update when the state auto updates

Reply:
Yeah great call. Totally forgot about selectors